src/pages/AdminManageDarCollections.js

```diff
@@ -38,8 +38,8 @@ export default function AdminManageDarCollections() {
   }, []);
 
   const updateCollections = updateCollectionFn({collections, filterFn, searchRef, setCollections, setFilteredList});
-  const cancelCollection = cancelCollectionFn({updateCollections, role: 'admin'});
-  const openCollection = openCollectionFn({updateCollections});
+  const cancelCollection = cancelCollectionFn({updateCollections, role: 'Admin'});
+  const openCollection = openCollectionFn({updateCollections, role: 'Admin'});
```

Comment:
nit: `USER_ROLES.admin` from `utils.js` might be a useful replacement for `role: 'Admin'`